spring-integration-file/src/main/java/org/springframework/integration/file/remote/synchronizer/AbstractInboundFileSynchronizer.java

```diff
@@ -165,6 +168,10 @@ public void setRemoteDirectoryExpression(Expression remoteDirectoryExpression) {
 	 * @param filter the file list filter.
 	 */
 	public void setFilter(FileListFilter<F> filter) {
+		doSetFilter(filter);
+	}
+
+	public void doSetFilter(FileListFilter<F> filter) {
```

Comment:
Doh. Yeah, really `protected final`.
That was already some time ago when this PR has been fresh in my mind. 😄